### DISCUSSION TITLE
Timeseries experiment

### DISCUSSION THREAD

**Discussion by Spongillas** [pycaret/pycaret#2707] [Q&A] [answered]
Hi!

An error is occured when I tried to import TimeSeriesExperiment:
`from pycaret.internal.pycaret_experiment import TimeSeriesExperiment`

ImportError: cannot import name 'TimeSeriesExperiment' from 'pycaret.internal.pycaret_experiment' (/home/skorohodov/Forecast/venv/lib/python3.8/site-packages/pycaret/internal/pycaret_experiment/__init__.py)

module version:
pycaret-ts-alpha==3.0.0.dev1649017462

**Comment by ngupta23** [comment, marked as answer]
This is the old format. Please check the notebooks in the GitHub repo for the new format for import.

Also you should consider upgrading to pycaret 3.0.0rc release which has updates to time series and time series merged with other modules.

`pip install -U --pre pycaret`

**Reply by Spongillas** [reply]
Thanks, I've found nothing in documentation. 
from pycaret.time_series.forecasting.oop import TSForecastingExperiment - is it correct?

**Reply by ngupta23** [reply]
Yes. It should be there in the notebooks.
